src/modules/project/pages1/projectDashboard/views/ProjectDashboardNotifications.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack, VStack } from '@chakra-ui/react'
+
+import { CreatorNotifications } from '@/modules/profile/pages/profileSettings/components/CreatorNotifications'
+import { Body } from '@/shared/components/typography'
+
+import { DashboardLayout } from '../common'
+
+export const ProjectDashboardNotifications = () => {
+  return (
+    <DashboardLayout width="full">
+      <VStack w="100%" spacing={6} flexGrow={1} px={{ base: 0, lg: 6 }}>
+        <VStack w="100%" alignItems="flex-start">
+          <Body size="lg" bold>
+            Notifications
+          </Body>
+          <HStack w="100%" justifyContent="space-between">
+            <Body>Customize your profile and project notifications.</Body>
+          </HStack>
```

Comment:
Not sure if the extra parent node is necessary if there is only one child.

```suggestion
            <Body>Customize your profile and project notifications.</Body>
```